proto/viam/common/v1/common.proto

```diff
@@ -65,6 +65,12 @@ message Orientation {
   double theta = 4;
 }
 
+message StaticFrame {
```

Comment:
Should this have a comment describing its purpose?

Reply:
Yes, definitely; I will add.